projects/uitdatabank/reference/entry.json

```diff
@@ -3272,6 +3272,115 @@
         }
       }
     },
+    "/images": {
+      "post": {
+        "summary": "image - create",
+        "operationId": "image-post",
+        "tags": [
+          "Images"
+        ],
+        "description": "Creates an image, which you can later link to one or more events, places, and organizers. Only images up to 21 million bytes (~ 20 MB) are allowed. The expected image format is `.png`, `.jpeg` or `.gif`.",
+        "requestBody": {
+          "description": "The image and the metadata to create.",
+          "content": {
+            "multipart/form-data": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "file": {
+                    "type": "string",
```

Comment:
To indicate that this needs to be a binary file, it should also have `"format": "binary"` according to https://swagger.io/docs/specification/describing-request-body/file-upload/ (See "Upload via Multipart Requests")